### DISCUSSION TITLE
Still getting Codespace Prebuild Usage even though prebuilds have been disabled<title>

### DISCUSSION THREAD

**Discussion by EdisonPhua** [community/community#48457] [Codespaces]
### Select Topic Area

Bug

### Body

Hi in my codespace settings, I disabled the Prebuild configuration about 1-2 weeks ago, However, I recently got an email couple of days ago saying I'm using up my free codespace storage and it says 12 GB is being used by Prebuild Storage. 

It appears I am having to same issue as the link below.
https://github.com/orgs/community/discussions/48213

**Comment by stauffenberg2020** [comment]
I am having the same issue. Lets see if Github staff have time to look into our issue
